lib/src/document.dart

```diff
@@ -84,6 +87,41 @@ class Document {
       }
     }
   }
+
+  static Element _classElement(
```

Comment:
This name is a bit odd; maybe something like `_elementWithClass`.

Alternatively, this helper may not be needed with my suggestions below; `Element(tag, children)..attributes['class'] = foo` should be fine to read.